File: src/main/java/afm/user/Theme.java
package afm.user;

import javafx.scene.layout.Pane;

import lombok.Getter;

// atm no Theme is applied to welcomeScreen
public enum Theme {
	DEFAULT("application.css"),
	LIGHT("lighttheme.css"),
	;

	@Getter
	final String stylesheet;

	Theme(String fileName) {
		this.stylesheet = "view/stylesheets/" + fileName;
	}

	@Override
	public String toString() {
		String name = this.name();
		return name.charAt(0) + name.substring(1).toLowerCase();
	}

	public void apply(Pane pane) {
		pane.getStylesheets().set(0, stylesheet);
	}
}

File: app/src/main/java/afm/user/Theme.kt
package afm.user

import javafx.scene.layout.Pane

// atm no Theme is applied to welcomeScreen
@Suppress("unused")
enum class Theme(fileName: String) {
    DEFAULT("application.css"),
    LIGHT("lighttheme.css");

    private val stylesheet = "view/stylesheets/$fileName"

    override fun toString(): String = name[0] + name.substring(1).lowercase()

    fun applyTo(pane: Pane) {
        pane.stylesheets[0] = stylesheet
    }
}
